task_reactredux/task1/src/Constants.js

```diff
@@ -6,3 +6,12 @@ const commandsEnum = Object.freeze({
 }); 
 
 export default (commandsEnum);
+
+export const MIN_PASSWORD_LENGTH = 5;
+
+export const REG_EXPR = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+
+export const ERROR_MESSAGES = [
```

Comment:
make this object with properties `EmailErrorMessage` and `PasswordErrorMessage`. Freeze this object, and use properties to get message like `errorMessagesEnum.EmailErrorMessage`